libzeth/circuits/merkle_tree/merkle_path_selector.hpp

```diff
@@ -69,6 +69,6 @@ class merkle_path_selector : public libsnark::gadget<FieldT>
 };
 
 } // namespace libzeth
-#include "merkle_path_selector.tcc"
+#include <libzeth/circuits/merkle_tree/merkle_path_selector.tcc>
```

Comment:
May be worth switching to `""` for all these local includes.  In fact, the original `"merkle_path_selector.tcc"` form could be used, since we are including another include in the same directory but may be more important to be consistent than to worry about the tiny cost saving.

Reply:
Indeed, must have done something wrong with my script (see comment below).
Should be addressed now. Thanks